health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/ActivityCard.js

```
@@ -0,0 +1,45 @@
+            style={{ 
+                minWidth: "100px", 
+                cursor: i.disable ? "not-allowed" : "pointer", 
+                opacity: i.disable ? 0.6 : 1,  // To visually indicate the disabled state
+                margin: "10px" 
+              }} // Adding margin to each card
+            onClick={() => {
+                if (!i.disable) { // Check if disabled
+                  i.link ? history.push(i.link) : location.assign(i.locate);
+                }
+              }}
+            children={
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)